README.md

```diff
@@ -181,6 +181,19 @@ File and folder paths inside `.gitattributes` are calculated relative to the pos
 *.rb linguist-language=Java
 ```
 
+Be aware that `gitattribute` values cannot contain whitespace.
+If a language's name contains spaces, replace them with hyphens instead:
+
+```gitattributes
+# These lines won't work:
+*.glyphs linguist-language=OpenStep Property List
+*.glyphs linguist-language=OpenStep\ Property\ List
+*.glyphs linguist-language="OpenStep Property List"
```

Comment:
I would remove these. The README is already longer than it should, and these examples are likely to be confusing for anyone who's skimming through.

To be honest, I would even prefer simply adding a new line to the above `=Java` example, with a comment that says `Replace whitespaces with hyphens`.

Reply:
Ooops. Eager beaver over here merged to soon 😊

Reply:
Ah yeah, true.

This is what happens when I proof-read my own writing too many times: I develop tunnel-vision. 😁

Reply:
I'm too slow these days...

Reply:
@pchaigno So, like this?

> ```gitattributes
> # Example of a `.gitattributes` file which reclassifies `.rb` files as Java:
> *.rb linguist-language=Java
>
> # Replace any whitespace in the language name with hyphens.
> ```

Or this?

> ```gitattributes
> # Example of a `.gitattributes` file which reclassifies `.rb` files as Java:
> *.rb linguist-language=Java
> ```
>
> Replace any whitespace in the language name with hyphens.

Reply:
Hm, too slow and unclear :)

I was thinking of something like this:
```gitattributes
# Example of a `.gitattributes` file which reclassifies `.rb` files as Java:
*.rb linguist-language=Java

# Replace any whitespace in the language name with hyphens:
*.glyphs linguist-language=OpenStep-Property-List 
```
When I'm skimming through documentation, I usually look only at the code/config. examples, barely reading any text. It might not be how other people read it though.